.changeset/odd-paws-reply.md

```diff
@@ -0,0 +1,6 @@
+---
+'@primer/react-brand': patch
+---
+
+- Improved `VideoPlayer` appearance on narrow viewports.
+- Removed `VideoPlayer` `showControlsWhenPaused` prop.
```

Comment:
Just curious, did we need to remove this as part of the accessibility update. Described as a UX improvement in PR description, but would like to understand how we comm this to users.

Reply:
I didn't strictly need to remove it, but I think the component is better for not having it. In general I think limiting the modification of video controls (closed captions, volume, audio tracks when we implement them down the line) to only when the video is playing could be detrimental to some users.

How about

```md

- Improved `VideoPlayer` appearance on narrow viewports.
  - This necessitated the removal of the `showControlsWhenPaused` prop to prevent the height of the `VideoPlayer` from changing when the video is played/paused.
```

Reply:
👍 LGTM, thanks